bench/commands/setup.py

```diff
@@ -135,23 +135,26 @@ def setup_socketio():
 @click.option("--node", help="Update only Node packages", default=False, is_flag=True)
 @click.option("--python", help="Update only Python packages", default=False, is_flag=True)
 @click.option("--dev", help="Install optional python development dependencies", default=False, is_flag=True)
-def setup_requirements(node=False, python=False, dev=False):
+@click.option("--app", help="setup requirements for a specific app")
```

Comment:
Setting up requirements for select apps would be cool...Also, we have similar patterns across Frappe IIRC. I think we take comma-separated inputs like "erpnext, healthcare, frappe"

Would be neater if we could send in a list of apps?


```suggestion
@click.option("--app", help="Setup requirements for a specific app")
```